swan-lake/learn/calling-java-code-from-ballerina.md

```diff
@@ -616,6 +611,15 @@ artifactId = "<artifact-id>"
 version = "<version>"
 ```
 
+If you wish to use a custom Maven repository, you can specify it in the `Ballerina.toml` file as shown below.
+```toml
+[[platform.java11.repository]]
+id = "<maven-repository-id>"
+url = "<maven-repository-url>"
+username = "<github-username>"
```

Comment:
@IrushiL This could be any maven repository where user should be authenticated. Github packages is one such repository.